internal/generator/generator.go

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) F5, Inc.
+ *
+ * This source code is licensed under the Apache License, Version 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package generator
+
+import (
+	"os"
+)
+
+func Generate(sourcePath string) error {
+	return genSupFromSrcCode(sourcePath, "directives", "Match", os.Stdout)
+}
```

Comment:
I think this would be more useful if implemented like this,

```go
// Generate... fill in missing doc comment
func Generate(w io.Writer, path string) error
```

This way any `io.Writer` can be passed. `cmd/generate/main.go` could pass `os.Stdout` and you can pass `bytes.Buffer` in tests. You could even pass an `http.ResponseWriter` if you wanted to return the result to an http request for some reason.

Reply:
Fixed: applied `func Generate(w io.Writer, path string) error`